### PR TITLE
Add Google Home dashboards

### PR DESCRIPTION
**Pull request by SamiSousa**

### CODE REVIEW

File: dashboards/google-home-analytics/metadata.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
+sample_dashboards:
+  -
+    category: Google Home
+    id: google_home_analytics
```
Comment (EvanSimpson) [inline dashboards/google-home-analytics/metadata.yaml:4]: These ids and file names need to be aligned around either hyphens or underscores, but they should match. Precedent in prior dashboards leans toward hyphens.

File: dashboards/google-home-analytics/google-home-analytics.01.png
Comment (EvanSimpson) [inline dashboards/google-home-analytics/google-home-analytics.01.png:1]: Are the additional screenshots for each of these dashboards intentionally omitted?

File: dashboards/google-home-analytics/google-home-analytics.json [hunk 2]
```diff
@@ -0,0 +1,742 @@
...
+              "scale": "LINEAR"
+            },
+            "chartOptions": {
+              "mode": "COLOR",
+              "showLegend": false,
+              "displayHorizontal": false
+            }
+          },
+          "title": " Fulfillment Request Count",
+          "id": ""
```
Comment (EvanSimpson) [inline dashboards/google-home-analytics/google-home-analytics.json:91]: Can we remove all of the vestigial empty `id` fields? I get the sense they should be specific to an instance of a dashboard, so don't really belong in a template.

Here and on other dashboards.

File: dashboards/google-home-analytics/google-home-analytics.json [hunk 3]
```diff
@@ -0,0 +1,742 @@
...
+          },
+          "title": "Daily Active Users - Device Type Breakdown",
+          "id": ""
+        }
+      }
+    ]
+  },
+  "dashboardFilters": [],
+  "labels": {}
+}
```
Comment (EvanSimpson) [inline dashboards/google-home-analytics/google-home-analytics.json:742]:
```suggestion
}
```

Let's remove these empty fields, here and on other dashboards.

**Review by EvanSimpson — CHANGES_REQUESTED** [review]

File: dashboards/google-home-analytics/google-home-analytics.01.png
Comment (SamiSousa) [reply dashboards/google-home-analytics/google-home-analytics.01.png:1]: Yes they are omitted. These screenshots provide the same coverage as the screenshots on the system dashboards for Google Home. I contacted the original contributors of those dashboards, and they did not provide me with a project with real data for those dashboards, and apparently they themselves did not have access to such a project.

The provided screenshots offer a base overview of what kind of charts each dashboard offer.

I don't think there's much value in showing multiple screenshots of fake data for each of these dashboards.

File: dashboards/google-home-analytics/google-home-analytics.json (hunk 2 above)
Comment (SamiSousa) [reply dashboards/google-home-analytics/google-home-analytics.json:91]: Done!

File: dashboards/google-home-analytics/metadata.yaml (hunk 1 above)
Comment (SamiSousa) [reply dashboards/google-home-analytics/metadata.yaml:4]: Updated!

File: dashboards/google-home-analytics/google-home-analytics.json (hunk 3 above)
Comment (SamiSousa) [reply dashboards/google-home-analytics/google-home-analytics.json:742]: Done!

**Review by EvanSimpson — APPROVED** [review]

**Pull request merged**